Practica2/includes/formularios/formulario.php

```diff
@@ -0,0 +1,148 @@
+<?php
+namespace SW_G5\includes\formularios;
+
+/**
+ * Clase base abstracta para la gestión de formularios.
+ * Define la estructura y lógica base para crear, procesar y validar formularios.
+ */
+abstract class formulario
```

Comment:
Ojo que aquí formulario está en minúsculas pero en los formularios usáis Formulario